src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -555,7 +555,7 @@ def makeDagSpecs(self, task, sitead, siteinfo, jobgroup, block, availablesites,
         dagSpecs = []
         i = startjobid
         temp_dest, dest = makeLFNPrefixes(task)
-        if task['tm_publication'] :
+	if task['tm_publication'] == 'T' :
```

Comment:
Ok, nevermind about the first space, it's a tab that shouldn't be there, please use spaces instead of tabs.